### PR TITLE
[CDAP-5291] [CDAP-6133] fix centering of flow diagram and workflow tokens

### PR DESCRIPTION
**Pull request by elfenheart**
Fix centering long flow diagram
Fix Workflow Tokens Icon appearing when program is not completed

JIRAs:
- CDAP-5291
- CDAP-6133

### CODE REVIEW

File: cdap-ui/app/directives/flow-graph/flow.js [hunk 1]
```diff
@@ -81,16 +81,18 @@ module.directive('myFlowGraph', function ($filter, $state, myStreamService, $loc
 
           // Pushing labels down
           parent.select('.label')
-            .attr('transform', 'translate(0,'+ bbox.height / 3 + ')')
+            .attr('transform', 'translate(20,'+ bbox.height / 3 + ')')
```
Comment (ajainarayanan) [inline cdap-ui/app/directives/flow-graph/flow.js:86]: We should probably add a small comment as to say why this was changed to `20`

**Comment by ajainarayanan** [comment]
I am seeing this in graph (the arrows going inside the nodes). Not sure if its a regression based on this change or is independent. 

screen shot 2016-06-07 at 10 33 37 am

**Comment by ajainarayanan** [comment]
One additional thing that I noticed is that we have a click handler on the token text 

```
if (isWorkflow) {
  scope.svg
   .selectAll('text.token-label')
   .on('click', scope.toggleToken);
}
```

If we could add the click event on the `g` element that wraps the `circle` and the `text` then it will be easier for the user to click and see the tokens. Right now I need to focus on the `T` inside the circle and click it. Initially I was under the assumption that the `circle` was also clickable 

screen shot 2016-06-07 at 10 40 40 am

**Comment by ajainarayanan** [comment]
👍 LGTM

**Comment by elfenheart** [comment]
demoed to @bdmogal

**Pull request merged**